linkerd/core/src/main/scala/io/buoyant/linkerd/Router.scala

```diff
@@ -122,13 +123,21 @@ trait RouterConfig {
   def interpreter: InterpreterConfig =
     _interpreter.getOrElse(defaultInterpreter)
 
+  @JsonProperty("bindingTimeoutMs")
+  var _bindingTimeoutMs: Option[Int] = None
+
+  @JsonIgnore
+  def bindingTimeout = _bindingTimeoutMs.map(_.millis).getOrElse(10.seconds)
```

Comment:
should this default to `timeoutMs`?  It doesn't make sense for this to be higher than the request timeout.